app/views/teacher_interface/further_information_request_items/edit.html.erb

```diff
@@ -17,5 +17,26 @@
     <p class="govuk-body">You can upload this document on the next screen.</p>
   <% end %>
 
+  <% if @further_information_request_item.work_history_contact? %>
+    <% @work_history = @further_information_request_item.work_history %>
+      <p class="govuk-body">Enter the contact details of a suitable referee who can verify your time spent working at <%= @work_history.school_name %> from <%= @work_history.start_date.to_fs(:month_and_year) %> <% if @work_history.end_date.present? %>
+          to <%= @work_history.end_date.to_fs(:month_and_year) %>
+        <% end %></p>
+      <%= f.govuk_text_field :contact_name %>
+
+      <%= f.govuk_text_field :contact_job %>
+
+      <%= f.govuk_email_field :contact_email %>
+
+      <%= govuk_details(summary_text: "Choosing someone to verify this role") do %>
+
+        <p>A senior member of staff means either a headteacher/principal, an assistant/deputy head or a school business manager.</p>
+
+        <p>Make sure you have an up-to-date email address for the person you choose, because the reference they provide will be important to your application.</p>
+
+        <p>We’ll ask them to confirm that you worked at the school for the period you’ve told us about.</p>
+      <% end %>
+  <% end %>
+
```

Comment:
I think the indentation in this file is a little off, could we look at correcting that?